starfish/pipeline/target_assignment/point_in_poly.py

```diff
@@ -1,41 +1,73 @@
 import numpy as np
-from skimage.measure import points_in_poly
 import pandas as pd
+import regional
+from skimage.measure import points_in_poly
+from tqdm import tqdm
 
 from starfish.constants import Features
+from starfish.intensity_table import IntensityTable
 from ._base import TargetAssignmentAlgorithm
 
 
 class PointInPoly(TargetAssignmentAlgorithm):
```

Comment:
Rename to PointInPoly2D to call out that this only works in 2D? Or at least, I think it only works in 2D.